examples/imagenet/requirements.txt

```diff
@@ -1,4 +1,5 @@
 clu==0.0.1a2
 ml-collections>=0.1.0
+optax
```

Comment:
Maybe specify version that includes `optax.linear_schedule()`?

Reply:
Done